DecipherNow/gm-fabric-dashboard
# feat: add storybook-intl

Resolves #1475 

## Proposed Changes

  - added storybook-addon-intl
  - added spanish translation for tooltip

## How to test
  - run storybook, look at Table > Instance Table. Switch to en-ES and `cmd-shift-F` to see the full tooltip.
